types/leaflet-draw/index.d.ts

```diff
@@ -167,7 +167,7 @@ declare module 'leaflet' {
              *
              * Default value: null
              */
-            remove?: null | false;
+            remove?: any;
```

Comment:
if this allows `true` now, why not
```suggestion
            remove?: boolean | null;
```

Reply:
From the code, it seems like the data is expected to be an object, but sure boolean will do too.